### PR TITLE
Add a soviet tanker helmet to the game.

### PR DESCRIPTION
**Pull request by BalthazarArgall**
#### Summary
Content "Add a soviet tanker helmet"

#### Purpose of change
The Tsh-4 helmet is a very common item to be sold in military surplus shops yet it doesn't exist in CDDA.

#### Describe the solution
I added it into the game.

#### Describe alternatives you've considered
Not adding it into the game.

#### Testing
I spawned the item in game and wore it without issues.

#### Additional context
It is based off the Tsh-4 helmet without being a specific variant of it.
This PR is mainly an opportunity for me to experience the merging process for myself.

### CODE REVIEW

File: data/json/items/armor/helmets.json [hunk 1]
```diff
@@ -306,6 +306,36 @@
...
+    "flags": [ "VARSIZE", "STURDY", "PADDED" ],
+    "armor": [
+      {
+        "layers": [ "NORMAL" ],
+        "encumbrance": 10,
+        "covers": [ "head" ],
+        "specifically_covers": [ "head_crown", "head_forehead", "head_ear_l", "head_ear_r" ],
+        "coverage": 100,
+        "material": [
+          { "type": "canvas", "thickness": 1 },
```
Comment (Maleclypse) [inline data/json/items/armor/helmets.json:332]: you have two canvas and no steel. Is that intended?

Comment (BalthazarArgall) [reply data/json/items/armor/helmets.json:332]: Yes, the second canvas is meant to represent the foam "casings", and the absence of steel is also intended, this is a soft helmet and unless I'm mistaken there is no steel plating in it.

Comment (BlizzDA) [reply data/json/items/armor/helmets.json:332]: The TSh-4 is a soft, padded leather/canvas (on later models) helmet. Doesn't contain steel, but were meant to protect from the tank's steel interior. The description is from `helmet_army_outdated` and probably an oversight.

Comment (Maleclypse) [reply data/json/items/armor/helmets.json:332]: Ok so then the description needs to stop saying "simple steel"
```
"A simple helmet meant to protect the head from fragmentation and impacts.  Its simple steel construction renders it outdated by modern military standards.",
```

Comment (BalthazarArgall) [reply data/json/items/armor/helmets.json:332]: I should have went to sleep **before** posting this looks like. Give me a moment.

Comment (BalthazarArgall) [reply data/json/items/armor/helmets.json:332]: > Ok so then the description needs to stop saying "simple steel"
> 
> ```
> "A simple helmet meant to protect the head from fragmentation and impacts.  Its simple steel construction renders it outdated by modern military standards.",
> ```

There you go, I updated the description to what it should be. Do tell me if the formulation feels weird.

File: data/json/itemgroups/Locations_MapExtras/locations_commercial.json [hunk 2]
```diff
@@ -414,6 +414,7 @@
       [ "coat_lab", 20 ],
       [ "helmet_army", 40 ],
       [ "helmet_army_outdated", 30 ],
+      [ "helmet_soviet_tanker", 30 ],
```
Comment (ANickelN) [inline data/json/itemgroups/Locations_MapExtras/locations_commercial.json:417]:
```suggestion
      [ "helmet_soviet_tanker", 3 ],
```
Cut down to 10% of chance of a WW2 US Helmet

File: data/json/itemgroups/Locations_MapExtras/locations_commercial.json [hunk 3]
```diff
@@ -758,6 +759,7 @@
       [ "helmet_liner", 20 ],
       [ "jacket_army", 40 ],
       [ "helmet_army_outdated", 10 ],
+      [ "helmet_soviet_tanker", 10 ],
```
Comment (ANickelN) [inline data/json/itemgroups/Locations_MapExtras/locations_commercial.json:762]:
```suggestion
      [ "helmet_soviet_tanker", 1 ],
```
Cut down to 10% of chance of a WW2 US Helmet

File: data/json/itemgroups/art_antiques_crafts.json [hunk 4]
```diff
@@ -206,6 +206,7 @@
     "subtype": "distribution",
     "entries": [
       { "item": "helmet_army_outdated", "prob": 5 },
+      { "item": "helmet_soviet_tanker", "prob": 5 },
```
Comment (ANickelN) [inline data/json/itemgroups/art_antiques_crafts.json:209]:
```suggestion
      { "item": "helmet_soviet_tanker", "prob": 2 },
```
Reduced by about half here.

File: data/json/items/armor/helmets.json [hunk 5]
```diff
@@ -306,6 +306,36 @@
     ],
     "melee_damage": { "bash": 2 }
   },
+  {
+    "id": "helmet_soviet_tanker",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "soviet tanker helmet" },
+    "description": "A soviet era tanker helmet.  It's distinctive foam pads are meant to protect a vehicle driver from bumps against the cramped interior.",
```
Comment (ANickelN) [inline data/json/items/armor/helmets.json:314]:
```suggestion
    "description": "A Soviet era tanker helmet.  It's distinctive foam pads are meant to protect a tank's driver from bumps against the cramped interior.",
```

**Review by ANickelN — COMMENTED** [review]
I think changing these values a bit should make the helmet fit in a bit more in New England.

File: data/json/items/armor/helmets.json (hunk 5 above)
Comment (BalthazarArgall) [reply data/json/items/armor/helmets.json:314]: Sure, sounds good to me. Is the description change mandatory or just a suggestion though?

Comment (ANickelN) [reply data/json/items/armor/helmets.json:314]: Not mandatory and I could be wrong.  Considering Soviet is short for Soviet Union I thought it should probably be capitalized too.

Comment (BalthazarArgall) [reply data/json/items/armor/helmets.json:314]: I'll keep Soviet capitalized, I think it's correct but the replacement of vehicle by tank is a good suggestion.

**Pull request merged**